kafka-rest/src/test/java/io/confluent/kafkarest/v2/KafkaConsumerManagerTest.java

```diff
@@ -50,10 +49,7 @@
 import io.confluent.rest.RestConfigException;
 import org.junit.runner.RunWith;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
Do we care if we `import *` in tests? This wasn't intentional but I now see that other test classes have that as well

Reply:
Just my personal preference, but I like tests to follow the same conventions as regular code. Not sure why we don't enforce checkstyle on test cases...